services/app-api/src/postgres/questionAnswers/insertQuestion.ts

```diff
@@ -0,0 +1,62 @@
+import { PrismaClient } from '@prisma/client'
+import { CMSUserType } from '../../domain-models'
+import { convertPrismaErrorToStoreError, StoreError } from '../storeError'
+import { Question, CreateQuestionInput } from '../../domain-models'
+import { v4 as uuidv4 } from 'uuid'
+
+export async function insertQuestion(
+    client: PrismaClient,
+    questionInput: CreateQuestionInput,
+    user: CMSUserType
+): Promise<Question | StoreError> {
+    const questionData = {
```

Comment:
Seems fine to box this up into its own variable, but not required. Was this from an earlier attempt at just passing the object into create()?

Reply:
I don't remember exactly what I was trying to do. I think it may have been left over from a previous version where `insertQuestion` calls another function to do the actual store and that was the object being passed in. Rolled all of that into `insertQuestion`, so that variable is not needed. Although I will keep the question mapping as a variable instead of putting it in the query.